### PR TITLE
fix #1778, AppSubmitter "-config" option is not used

### LINKED ISSUE #1778: AppSubmitter "-config" option is not used
We already have a "-conf" option on Gear which allows users to set a custom config file.

### PR DESCRIPTION
**Pull request by clockfly**

### CODE REVIEW

**Comment by manuzhang** [comment]
Could you add "-conf" options to doc(https://github.com/gearpump/gearpump/blob/master/docs/commandline.md) as well ?

**Comment by clockfly** [comment]
@manuzhang 
 updated.

File: docs/commandline.md [hunk 1]
```diff
@@ -80,5 +80,5 @@ Please check [Deployment for Standalone mode](deployment-standalone.html) for mo
 To start UI server, you can:
 
 ```bash
-services
+services  [-conf <custom gearpump config file>]
```
Comment (manuzhang) [inline docs/commandline.md]: services has `config` option. Please check https://github.com/gearpump/gearpump/blob/master/services/jvm/src/main/scala/io/gearpump/services/main/Services.scala#L35

**Comment by clockfly** [comment]
Ok, done.

Some default options are not listed, since I think we don't need to document everything.

**Comment by manuzhang** [comment]
+1

**Pull request merged**